From 71afd48a794db25f3cfd0f278e06b2917799f4a6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Arthur=20Exalta=C3=A7=C3=A3o?=
 <arthur.exaltacao@chainguard.dev>
Date: Wed, 13 Aug 2025 01:06:34 -0300
Subject: [PATCH] samba: use correct directory for searching for libidmap
 private samba libraries
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

In wolfi's samba package we seem to be installing that library to
`/usr/lib` instead of `/usr/lib/samba`, which makes this test error out

Signed-off-by: Arthur Exaltação <arthur.exaltacao@chainguard.dev>
---
 src/external/samba.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/external/samba.m4 b/src/external/samba.m4
index 06804c130..cad1f5fdc 100644
--- a/src/external/samba.m4
+++ b/src/external/samba.m4
@@ -55,7 +55,7 @@ with argument --without-samba
     else
         sambalibdir="`$PKG_CONFIG --variable=libdir smbclient`"/samba
         AC_MSG_CHECKING([Samba's idmap library])
-        if test -f "${sambalibdir}/libidmap-private-samba.so"; then
+        if test -f "/usr/lib/libidmap-private-samba.so"; then
                 IDMAP_SAMBA_LIBS=idmap-private-samba
         elif test -f "${sambalibdir}/libidmap-samba4.so"; then
                 IDMAP_SAMBA_LIBS=idmap-samba4
-- 
2.50.1


